data/scenarios/Testing/490-harvest.yaml

```diff
@@ -1,15 +1,19 @@
 name: Test harvest command
-description: Test the difference between the grab and harvest commands.
+description: |
+  Test the difference between the grab and harvest commands,
+  and test that grab doesn't drain water.
+  https://github.com/swarm-game/swarm/issues/490
 win: |
   try {
     turn east; move;
     t1 <- ishere "tree";
     move;
     t2 <- ishere "tree";
-    return (t1 && not t2)
+    t3 <- as base {n <- count "water"; return (n == 4)};
+    return (t1 && not t2 && t3)
   } { return false }
 solution: |
-  move; harvest; move; grab
+  move; harvest; move; grab; turn right; move; grab; grab; grab; grab
```

Comment:
Could you make one of those a harvest?

I expect it makes no difference, but I would like to have the expected behaviour tested.